app/models/stripe_wrapper.rb

```diff
@@ -1,22 +1,65 @@
 module StripeWrapper
+  def self.set_api_key(key)
+    Stripe.api_key = key
```

Comment:
I would move this to its own initializer, so the key is set when your app starts.